scripts/prep_forecast.sh

```diff
@@ -281,7 +281,9 @@ sed -i -e "s;DT_DYNAM_MOM6;${DT_DYNAM_MOM6};g" tmp1
 # Rename to proper input ice input name
 mv tmp1 $DATA/INPUT/MOM_input
 
-cp $TEMPLATEDIR/MOM_override $DATA/INPUT/MOM_override
+#cp $TEMPLATEDIR/MOM_override $DATA/INPUT/MOM_override
```

Comment:
delete this line?

Reply:
I think this should be there?

Reply:
This is how we add river runoff to MOM6

Reply:
Good to know :dancing_men:

Reply:
so @guillaumevernieres why did you comment it out?

Reply:
@flampouris, the commented out line is not needed.
MOM_override is a file in which extra lines can be added pertaining to for example, riverrunoff, precip etc.

Reply:
@ShastriPaturi  you are correct, what is in the scripts/template/MOM6_override is currently blank.  In the DATM-MOM6-CICE5 we use the data_table not the MOM6-override capability to add river runoff.

Reply:
As the name indicate, MOM_override, overrides what's in MOM_input. 
It's not blank in this PR.
I should have left a comment where I commented out the copy from the template. I tried appending to MOM_override (>> instead of >,  [L285](https://github.com/NOAA-EMC/godas/blob/f8a2f4efb878d6c0f8a0b8ba053046ce7df2a915/scripts/prep_forecast.sh#L285)) but it failed, not all the time, not sure why. Suggestions appreciated, but since the original file is empty, I figured it's probably OK.